README.md

```
@@ -1,127 +1,177 @@
-summary_chain = PRSummaryChain.from_llm(code_summary_llm=llm35, pr_summary_llm=llm4, verbose=True)
-
-summary = summary_chain({"pull_request": retriever.pull_request}, include_run_info=True)
+## Configuration
 
-print(summary)
+Codedog uses environment variables for configuration. You can set these directly in your shell, or use a `.env` file (you might need to install `python-dotenv` separately in your environment: `poetry run pip install python-dotenv`).
+
+**Required:**
+
+*   **Platform Token**:
+    *   For GitHub: `GITHUB_TOKEN="your_github_personal_access_token"`
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)